common/djangoapps/track/backends/logger.py

```diff
@@ -33,7 +33,13 @@ def __init__(self, name, **kwargs):
         self.event_logger = logging.getLogger(name)
 
     def send(self, event):
-        event_str = json.dumps(event, cls=DateTimeJSONEncoder)
+        try:
+            event_str = json.dumps(event, cls=DateTimeJSONEncoder)
+        except UnicodeDecodeError:
+            for key in event.iterkeys():
+                if isinstance(event[key], str):
+                    event[key] = event[key].decode('latin1')
```

Comment:
@singingwolfboy , @brianhw using ```latin-1``` decoding here because characters described in the description of ticket are latin ones [AN-4509](https://openedx.atlassian.net/browse/AN-4509).
0xF1, 0xF3, and 0xE9. In latin-1, these would be n-tilde, o-acute, and e-acute.